src/server/MessageProcessor.js

```diff
@@ -0,0 +1,419 @@
+/**
+ *  Copyright (c) Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ *  @flow
+ */
+
+import type {Diagnostic, Uri} from '../types/Types';
+
+import {getGraphQLCache} from './GraphQLCache';
+
+import {getOutline} from '../interfaces/getOutline';
+import {GraphQLLanguageService} from '../interfaces/GraphQLLanguageService';
+
+import {findGraphQLConfigDir} from '../config/GraphQLConfig';
+
+import {Position} from '../utils/Range';
```

Comment:
Usually I tried to alphabetically sort imported modules, and group by type imports/named/local with the whitespaces in between. I think I went a bit insane for this diff though - I'll make sure to sort accordingly whenever I see it